File: src/frontend/launcherComponent/src/app/shared/model/obiettivo-sensibile.ts
import { Localita } from './localita.model';
import { Coordinate } from './coordinate.model';

export class ObiettivoSensibile extends Localita {
    constructor(
        public nome: string,
        public documenti: string[],
        public coordinate: Coordinate,
        public indirizzo?: string
    ) {
        super(coordinate, indirizzo);
    }
}

File: src/backend/SO115App.Models/Classi/Condivise/ObiettivoSensibile.cs
using SO115App.API.Models.Classi.Condivise;
using System;
using System.Collections.Generic;
using System.Text;

namespace SO115App.Models.Classi.Condivise
{
    public class ObiettivoSensibile
    {
        private string Nome { get; set; }
        private string[] Documenti { get; set; }
        private Coordinate Coordinate { get; set; }
        private string Indirizzo { get; set; }
    }
}
